src/components/ecl-button/ecl-button.stories.ts

```diff
@@ -9,21 +9,40 @@ const getArgs = () => {
     iconPosition: 'after',
     iconTransform: '',
     hideLabel: false,
+    indicator: false,
+    disabled: false,
+    indicatorValue: 3,
   };
 };
 
 const getArgTypes = () => {
   return {
+    color_mode: { table: { disable: true } },
     variant: {
       name: 'button variant',
       type: { name: 'select' },
-      options: ['primary', 'secondary', 'ghost', 'call'],
+      options: ['primary', 'secondary', 'ghost', 'cta', 'tertiary', 'ghost-inverted'],
+      control: {
+        labels: {
+          primary: 'Primary',
+          secondary: 'Secondary',
+          ghost: 'Ghost',
+          cta: 'Call to action',
+          tertiary: 'Tertiary',
+          'ghost-inverted': 'Ghost inverted',
+        },
+      },
       description: "Button variant"
     },
+    disabled: {
+      name: 'Disabled',
```

Comment:
it seems we are using lowercased labels for the controls, this stands out because it is capitalized